src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -56,4 +57,19 @@
       SourceFiles="$(_AndroidSdkLocation)\$(_SupportLicenseInAndroidSdk)"
       DestinationFiles="$(OutputPath)..\..\..\mandroid\MULTIDEX_JAR_LICENSE" />
   </Target>
+
+  <Target Name="_BuildMonoSymbolicate">
+    <MSBuild
+      Projects="..\..\external\mono\mcs\tools\mono-symbolicate\monosymbolicate.csproj"
```

Comment:
Style: attributes are indented an extra 2 spaces, so that they're indented more than any nested child elements.